pkg/router/template/plugin.go

```diff
@@ -311,6 +313,8 @@ func createRouterEndpoints(endpoints *kapi.Endpoints, excludeUDP bool, lookupSvc
 					Port: strconv.Itoa(int(p.Port)),
 
 					PortName: p.Name,
+
+					Idled: wasIdled,
```

Comment:
Could we just make this `NoHealthCheck` so the template doesn't have to know about it?